packages/manager/modules/pci/src/projects/project/storages/volume-backup/list/delete/controller.js

```diff
@@ -0,0 +1,50 @@
+export default class VolumeBackupDeleteController {
+  /* @ngInject */
+  constructor($translate, VolumeBackupService) {
+    this.$translate = $translate;
+    this.volumeBackupService = VolumeBackupService;
+
+    this.DELETE_CONFIRMATION_INPUT = /^DELETE$/;
```

Comment:
**nitpick:** Already defined in `packages/manager/modules/pci/src/projects/project/storages/volume-backup/list/delete/constants.js`. Re-use

Reply:
forget to remove the constants file.
No more required.

done.